server/src/test/java/io/crate/action/sql/CursorTest.java

```diff
@@ -322,5 +432,33 @@ public void test_scrolling() throws Exception {
                 2
                 """
         );
+
+        assertThat(result).isNotCompleted();
+        cursor.close();
+        assertThat(result).isCompleted();
```

Comment:
Wouldn't it be enough if we assert this at 1 dedicated test case instead of everywhere?

Reply:
It is, but I think it doesn't hurt to have some extra assertions in a unit test.